pkg/kubectl/cmd/exec.go

```diff
@@ -48,6 +48,10 @@ var (
 		kubectl exec 123456-7890 -c ruby-container -i -t -- bash -il`)
 )
 
+const (
+	execUsageStr = "expected 'exec POD_NAME COMMAND [ARG1] [ARG2] ... [ARGN]'.\nPOD_NAME and COMMAND are required arguments for the exec command"
```

Comment:
That's an option, but it's not required, it's only needed if you want to pass flags to the command that you execute `kubectl exec foo sh` works just fine.